internal/tenancy/internal/types/namespace.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package types
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/hashicorp/consul/internal/resource"
+	"github.com/hashicorp/consul/proto-public/pbresource"
+	tenancyv1alpha1 "github.com/hashicorp/consul/proto-public/pbtenancy/v1alpha1"
```

Comment:
Can we make an effort to not bake the `v1alpha1` in the names/references to things throughout the code unless absolutely necessary so that it is easier to refactor when we eventually bump the group version?

Reply:
yeah good call, I will rename those

Reply:
Actually this is generated by the proto, I mirrored what catalog was doing and created a sub package called v1alpha1 for the proto files. I'm not sure what is the plan with those and if it have any implication for future breaking changes.

Reply:
No worries, let's just roll with it for now.